pom.xml

```diff
@@ -1110,6 +1111,12 @@
                 <version>hadoop2-${dep.gcs.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
I don’t know why we’re adding all these dependencies, but something is wrong. Is this depending on a non-shaded version of Hadoop?

Reply:
@electrum yes its not using the shaded version of hadoop.  The lib uses  [these versions of hive/hadoop] (https://github.com/starburstdata/hive-json-serde/blob/develop/pom.xml#L41). @MiguelWeezardo have we tried using the shaded version of hadoop for this?

Reply:
I haven't tried shading hadoop yet.

Reply:
https://github.com/starburstdata/hive-json-serde/pull/8